proto/platform/oidc/v1/oidc.platform.proto

```diff
@@ -19,6 +19,26 @@ service SecurityTokenService {
       }
     };
   }
+
+  rpc GetAccessToken(GetAccessTokenRequest) returns (TokenPair) {
+    option (google.api.http) = {
+      post: "/sts/access_token"
+      additional_bindings {
+        get: "/sts/access_token"
+      }
+    };
+  }
+}
+
+message GetAccessTokenRequest {
+  repeated string aud = 1;
+  string scope = 2;
+  string refresh_token = 3;
+
+  // List of capabilities to request for the token.
+  repeated string cap = 4 [(google.api.resource_reference) = {
+    type: "chainguard.capabilities/Capability"
```

Comment:
Hmm, I'm not familiar with this syntax.  Why not a repeated enum field?  Did we change something about capabilities?

Reply:
This is copy/pasta from current `ExchangeRequest`